config/_default/menus/menus.en.yaml

```diff
@@ -1014,27 +1014,33 @@ main:
     identifier: "synthetics_browser_tests"
     weight: 2
 
```

Comment:
looks like browser tests got removed from the toc somehow? see preview

Reply:
fixed, it was an id issue well spotted !